### ISSUE TITLE
Don't throw error when unregistering unknown id

### ISSUE THREAD

**Issue by madvas** [day8/re-frame-forward-events-fx#4]
https://github.com/Day8/re-frame-forward-events-fx/blob/master/src/day8/re_frame/forward_events_fx.cljs#L14

Could we remove throwing error when unregistering unknown error? This can be pretty annoying, since there's many use cases when setting up forwarder is under some condition, but cleaning up function shouldn't need to reverse engineer those conditions. 

If you agree I'll be happy to do PR. 

Thank you, great library

**Comment by mike-thompson-day8** [comment]
Hard for me to imagine this usecase, could you elaborate?  

But, in any case, I'll take your word that a usecase exists.  :-)  So, yes, I'll take a PR, but how about we do it this (backwards compatible) way ... 

Add an optional way for `:unregister` to happen silently on failure. The default continues to be noisy (exception) failure, because I think that's still the best default action, but we add a way to silence this.

So the PR would make this effect possible:
```
{:forward-events  {:unregister      :the-id-supplied-when-registering
                   :silent-failure  true}}        ;; <---- this is new. defaults to false
```

**Comment by superstructor** [comment]
Fixed by [cee15ec](https://github.com/day8/re-frame-forward-events-fx/commit/cee15ec7401440dce68d8349a35c9b1e2ee6019f). Also see #5.
